src/style/themes/base/base-theme.config.js

```diff
@@ -272,6 +272,18 @@ export default (palette) => {
       errorButtonFocus: palette.errorRedShade(20),
     },
 
+    scrollbar: {
+      light: {
+        thumb: palette.slateTint(40),
+        track: palette.slateTint(90),
```

Comment:
Talked the Harpal and this should actually be `slate tint 95`

Reply:
I'll update the theme config while Nick focuses on the navigation work